spec/controllers/api/reservations_controller_spec.rb

```diff
@@ -83,4 +104,25 @@
 
     expect(response.status).to eq(403)
   end
+
+  def event_date_response
```

Comment:
You can simplify this by leaving off the unused fields. No need include things like `accept_reservations` here.